docs/feature_dynamic_macros.md

```diff
@@ -2,43 +2,45 @@
 
 QMK supports temporary macros created on the fly. We call these Dynamic Macros. They are defined by the user from the keyboard and are lost when the keyboard is unplugged or otherwise rebooted.
 
-You can store one or two macros and they may have a combined total of 128 keypresses. You can increase this size at the cost of RAM.
+You can store one or two dynamic macros and they may have a combined total of 64 key strokes (i.e., 128 key press or release events). You can increase this size at the cost of RAM, see [`DYNAMIC_MACRO_SIZE` below](#customization).
```

Comment:
A bit tangential, but this is correct on reviewing the code. I can split this into separate PR if you want